ngraph/python/CMakeLists.txt

```diff
@@ -29,8 +29,12 @@ if(NOT pybind11_POPULATED)
     add_subdirectory(${pybind11_SOURCE_DIR} ${pybind11_BINARY_DIR})
 endif()
 
-# PYTHON_VERSION_MAJOR and PYTHON_VERSION_MINOR are defined inside pybind11
-set(PYTHON_VERSION python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR})
+find_package(PythonLibs 3 REQUIRED)
+if(PYTHONLIBS_VERSION_STRING MATCHES "^([0-9]+)\.([0-9]+).*")
+    set(PYTHON_VERSION python${CMAKE_MATCH_1}.${CMAKE_MATCH_2})
+else()
+    message(FATAL_ERROR "Failed to extract python major.minor from ${PYTHONLIBS_VERSION_STRING}")
+endif()
```

Comment:
nitpick: can we introduce new common function to avoid code duplication?

Reply:
looks like all these places depend on `InferenceEngineDeveloperpackage` optionally and should be self-sufficient.